ui/arduino/store.js

```diff
@@ -15,7 +15,7 @@ function store(state, emitter) {
   const disk = window.BridgeDisk
   const win = window.BridgeWindow
 
-  win.setWindowSize(700, 640)
+  win.setWindowSize(720, 640)
```

Comment:
why do we need to set the window's size when it's already set in its initial `new BrowserWindow()` call?

Reply:
It's because the app sets a default global screen constraints and the UI can "request" new window constraints. In this case particularly it's the same value.